packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -1318,17 +1333,28 @@ export function TransferPanel() {
             const { estimatedL1GasFees, estimatedL2GasFees } =
               sanitizedEstimatedGasFees
 
-            // We have to check if there's enough ETH to cover L1 gas, and enough of the custom fee token to cover L2 gas
-            if (
-              estimatedL1GasFees > ethBalanceFloat ||
-              estimatedL2GasFees > customFeeTokenL1BalanceFloat
-            ) {
-              return TransferPanelMainErrorMessage.INSUFFICIENT_FUNDS
+            // We have to check if there's enough ETH to cover L1 gas
+            if (estimatedL1GasFees > ethBalanceFloat) {
+              return getInsufficientFundsForGasFeesErrorMessage({
+                asset: ether.symbol,
+                chain: sourceChain
+              })
+            }
+
+            // We have to check if there's enough of the custom fee token to cover L2 gas
+            if (estimatedL2GasFees > customFeeTokenL1BalanceFloat) {
+              return getInsufficientFundsForGasFeesErrorMessage({
+                asset: nativeCurrency.symbol,
+                chain: sourceChain
+              })
             }
           }
 
           if (defaultRequiredGasFees > ethBalanceFloat) {
```

Comment:
do we want to show a slightly different message for this?
Asking because `defaultRequiredGasFees =
          sanitizedEstimatedGasFees.estimatedL1GasFees +
          sanitizedEstimatedGasFees.estimatedL2GasFees`

Reply:
I wouldn't make it any more verbose. The user just needs to know that they don't have enough ETH to pay for gas, they don't need to know that it will be split between L1 and L2 (they can already see it in summary)